### ISSUE TITLE
Issues with world generation

### ISSUE THREAD

**Issue by datitisev** [scalda/Foolcraft_3#13]
There seem to be more issues with world generation than in Foolcraft & Foolcraft 2.
Just by exploring the world I found these weird issues (not visual):

![2018-05-19_17 03 22](https://user-images.githubusercontent.com/6401250/40279578-0c500dd8-5c13-11e8-8757-512735d2489e.png)
![2018-05-19_17 07 24](https://user-images.githubusercontent.com/6401250/40279580-0ee62abe-5c13-11e8-949f-0f8383d7b277.png)

The chunk one looks like a corrupted chunk, but I was able to get into the water and wasn't teleported back up because of "ghost blocks".

**Comment by Mrbysco** [comment]
The hole is the only thing I see as an issue. The other is something that can even happen in vanilla

**Comment by datitisev** [comment]
I know corruptions can occur in vanilla, but usually they are extremely rare. I found both of these in the 15 minutes I was exploring, just walking around.

I thought vanilla chunk corruptions were only visual, that if you went in them you'd get stuck because there were blocks there, and were fixed with a relog. This chunk was just all water in the middle of nowhere. Not sure :/

**Comment by Seegras** [comment]
You should specify which generator/world type you did use. Biomes O'Plenty or "default"?

**Comment by datitisev** [comment]
@Seegras Right, my apologies. I think I used the Biomes O’Plenty generator.... however I’m not sure. I’ll try to see if I can check that in a few hours later today
